packages/synapse-react-client/src/lib/utils/SynapseClient.ts

```diff
@@ -2859,11 +2860,12 @@ export const searchAccessRequirements = (
  * @param {string} requirementId id of entity to lookup
  * @returns {AccessRequirementStatus}
  */
-export const getAccessRequirementStatus = (
-  accessToken: string | undefined,
-  requirementId: string | number,
-): Promise<AccessRequirementStatus | ManagedACTAccessRequirementStatus> => {
-  return doGet(
+export function getAccessRequirementStatus<
```

Comment:
Add generic so we can be more confident about the return type if we know we're fetching status for a ManagedACTAccessRequirement